### PR TITLE
Add cup icon to web icons set

### PR DESCRIPTION
**Pull request by Malinicz**
image

### CODE REVIEW

**Review by matzimowski — APPROVED** [review]
🥤  🥇

File: src/images/icons/cup.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (Malinicz) [inline src/images/icons/cup.svg:1]: I removed `fill` from here (comparing to mobile icon), should I actually do that @coderitual?

**Review by bartoszlorek — APPROVED** [review]

File: src/images/icons/cup.svg (hunk 1 above)
Comment (coderitual) [reply src/images/icons/cup.svg:1]: great, we don't use `fill` one color icons

**Review by coderitual — APPROVED** [review]
✏️

**Pull request merged**